okapi-core/src/main/java/org/folio/okapi/managers/TenantManager.java

```diff
@@ -572,11 +573,11 @@ private void fireTimer(Tenant tenant, ModuleDescriptor md, RoutingEntry re, Stri
           "Bad _tenantPermissions interface in module " + permsModule.getId()
               + ". No path to POST to"));
     }
-    pc.debug("tenantPerms: " + permsModule.getId() + " and " + permPath);
+    logger.debug("tenantPerms: {} and {}", (Supplier<?>) () -> permsModule.getId(), permPath);
```

Comment:
Would this work `logger.debug("tenantPerms: {} and {}", permsModule.getId(), permPath);`?

Reply:
yes.